TESTS/usb_device/basic/USBTester.cpp

```diff
@@ -411,7 +437,11 @@ const uint8_t *USBTester::configuration_desc()
         1                           // bInterval
 
     };
-    return config_descriptor;
+    if (index == 0) {
+        return config_descriptor;
+    } else {
+        return NULL;
+    }
```

Comment:
I think this should be returned for the index 1 rather than 0 if you want to match bConfigurationValue.

Reply:
According to docs and my observation it's ok
Only the set_configuration request starts from 1 since 0 is reserved for deconfiguration

```
9.4.3 Get Descriptor
...
The range of values used for a descriptor index is from 0 to one less than the number
 of descriptors of that type implemented by the device.
...
```